test/downloads.jl

```diff
@@ -0,0 +1,16 @@
+using MarketData
+using Base.Test
+
+@testset "remote" begin
+    @testset "FRED" begin
+        ta = fred()
+        @test length(ta.timestamp) > 100
+    end
+
+    @testset "Yahoo" begin
+        # ta = yahoo()
+        # @test length(ta.timestamp) > 100
+        @test_broken 1==2
+    end
```

Comment:
so, remove this code block?

Reply:
No I prefer to keep it and mark this test as broken until `yahoo` function exists.
We could remove test block when/if `yahoo` function doesn't exist anymore.

Reply:
okay.